<!-- test-analysis-auto -->
@claude 以下のテスト失敗を修正してください。

**失敗ジョブ**: Build
**Workflow Run**: https://github.com/sy-hash/mind-echo/actions/runs/22727532068

```
Build	Build for Testing	2026-03-05T17:36:55.0654510Z /Users/runner/work/mind-echo/mind-echo/MindEcho/MindEcho/Views/RecordingModalView.swift:9:25: error: failed to produce diagnostic for expression; please submit a bug report (https://swift.org/contributing/#reporting-bugs)
```

テストが通るようにコードを修正し、ビルドが通ることも確認してください。

--- a/MindEcho/MindEcho/Services/VocabularyStore.swift
+++ b/MindEcho/MindEcho/Services/VocabularyStore.swift
@@ -25,7 +25,9 @@ final class VocabularyStore {
     }
 
     func remove(at offsets: IndexSet) {
-        words.remove(atOffsets: offsets)
+        for index in offsets.sorted().reversed() {
+            words.remove(at: index)
+        }
     }
 
     private func save() {

--- a/MindEcho/MindEcho/Views/RecordingModalView.swift
+++ b/MindEcho/MindEcho/Views/RecordingModalView.swift
@@ -138,7 +138,7 @@ struct RecordingModalView: View {
         }
         .onAppear {
             if ProcessInfo.processInfo.arguments.contains("--mock-transcription") {
-                viewModel.transcribe = { _, _ in
+                viewModel.transcribe = { _, _, _ in
                     try await Task.sleep(for: .milliseconds(500))
                     return "これはモックの書き起こし結果です。テスト用のテキストデータ。"
                 }

